mm/vmscan.c

```diff
@@ -146,7 +146,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 15;
```

Comment:
I don't think it plays well with our 3GB RAM devices. Anyway, changing default value directly in kernel used by range of different devices is not a very wise decision